### DISCUSSION TITLE
Why ernie_TIenvelope has no representation of electric field strength

### DISCUSSION THREAD

**Discussion by 784622315** [simnibs/simnibs#392] [9. Results and analysis]
Dear Professors, Hello, I am now using your team to develop the TI example simulation, and I successfully simulation results. But I encountered a problem, I simulation magn_E_1 and magn_E_2 can be successful display, but ernie_TIenvelope no matter what I can not display, the results are as follows, this problem has bothered me for a long time, i look forward to and appreciate your answer.
![007d5e520bbf20381a69f17e7bb3bc4](https://github.com/user-attachments/assets/93203d99-6fcc-4f67-85fd-e41ce937533c)
![ac7b6b3ae0c9c8237254af9b3bf5cce](https://github.com/user-attachments/assets/bac77089-ae32-4f66-99f3-419fc9f7fe2e)
![5f33ccbaaa39fe131f5bb1b99878e1e](https://github.com/user-attachments/assets/7fd26d2a-1d04-47a0-8912-f11aaecaf929)

**Comment by oulap** [comment]
Hi,

It looks like you might be visualizing the surface faces in gmsh while trying to display the simulation results. In this case the face "block" the field results and you just end up seeing the gray matter surface. Please have a look at this gmsh tutorial:

https://simnibs.github.io/simnibs/build/html/tutorial/visualization.html

specifically the "Selecting what to visualize"-section.

Hope that helps!
